### DISCUSSION TITLE
Webhook - new item on supressions list

### DISCUSSION THREAD

**Discussion by kluvi** [postalserver/postal#2199] [Feature suggestions]
It would be great, if we can configure webhook to notify about new recipient on supressions list.

**Comment by willpower232** [comment]
arguably you could listen for and act on all the MessageDeliveryFailed webhooks which should cover any problems you experience.

**Reply by kluvi** [reply]
I have not tested it yet, but description says `An e-mail cannot be delivered to its endpoint. This is a permanent failure so it will no be retried.`, which implies to me, that its "something with incoming emails routing". But I will give it a try.
